LibTest/ffi/Size/Size_A02_t01.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The C size_t type.
+///
+/// The [Size] type is a native type, and should not be constructed in Dart
+/// code. It occurs only in native type signatures and as annotation on [Struct]
+/// and [Union] fields.
+///
+/// @description Checks that this type represents C size_t type and is big
+/// enough to contain the size of the biggest object the host system can handle
+/// @author sgrekhov22@gmail.com
+
+import "dart:ffi";
+import "package:ffi/ffi.dart";
+import "../../../Utils/expect.dart";
+
+void main() {
+  Expect.equals(sizeOf<IntPtr>(), sizeOf<Size>());
```

Comment:
@dcharkes Is it correct to assume the above?

Reply:
On all platforms that we currently support yes.